modular_skyrat/master_files/code/modules/uplink/uplink_items.dm

```diff
@@ -233,12 +249,12 @@
 	item = /obj/item/ammo_box/magazine/m12g/slug
 
 //SUITS
-/datum/uplink_item/suits/hardsuit/elite_traitor
-	name = "Elite Syndicate Hardsuit"
-	desc = "An upgraded, elite version of the Syndicate hardsuit. It features fireproofing, and also \
-			provides the user with superior armor and mobility compared to the standard Syndicate hardsuit."
-	item = /obj/item/clothing/suit/space/hardsuit/syndi/elite
-	cost = 10
+///datum/uplink_item/suits/hardsuit/elite_traitor
+//	name = "Elite Syndicate Hardsuit"
+//	desc = "An upgraded, elite version of the Syndicate hardsuit. It features fireproofing, and also \
+//			provides the user with superior armor and mobility compared to the standard Syndicate hardsuit."
+//	item = /obj/item/clothing/suit/space/hardsuit/syndi/elite
+//	cost = 10
```

Comment:
This is not how you comment out something. Put a `/*` at the beginning and a `*/` at the end. Plus, this is a modular file so you can just delete it, no reason to leave it commented.

Reply:
Deleted it, same with robohand. The code for the item and bundle are still in the game, but I don't plan to bring them back so.